frontend/integration-tests-cypress/cypress-e2e/cypress/tests/monitoring_spec.ts

```diff
@@ -0,0 +1,89 @@
+import { testName, submitButton, errorMessage } from '../support/index';
+
+const testDetailsPage = (pageTitle: string, sectionHeading: string, testLabel = true) => {
+  cy.getTestElement('resource-title', 18000).contains(pageTitle); // long timeout for detail page to load
+  cy.getTestElement('section-heading').first().should('have.text', sectionHeading);
+  if(testLabel) {
+    cy.get('.co-m-label').first().contains(`alertname=${pageTitle}`);
+  }
+};
+
+describe('Monitoring: Alerts', () => {
+  // TODO: need to move these to before/after all specs run,
+  //  not just before/after this spec
+  before(() => {
+    cy.createTestProject(testName);
+  });
+  after(() => {
+    cy.deleteTestProject(testName);
+  });
+
+  it('displays the Alerts list page', () => {
+    cy.clickNavLink(['Monitoring', 'Alerting']);
+    cy.getTestElement('resource-title').should('have.text', 'Alert');  //purposefully fail test to see what it looks like in CI
+  });
+
+  it('does not have a namespace dropdown', () => {
+    cy.getTestElement('namespace-bar-dropdown').should('not.exist');
+  });
+
+  it('filters Alerts by name', () => {
+    cy.filterListPageByName('Watchdog');
+    cy.getTestElement('alert-resource-link')
+      .should('have.length', 1)
+      .should('have.text', 'Watchdog');
+  });
+
+  it('displays Alert details page', () => {
+    cy.getTestElement('alert-resource-link').first()
+    .should('have.text', 'Watchdog')
+    .click();
+    testDetailsPage('Watchdog', 'Alert Details');
```

Comment:
I'm inclined to name `testDetailsPage` something that implies asertion with pass/fail possibilities.

Reply:
Can you pls elaborate, give example of what that might be? -thanks